src/test/java/bio/terra/common/ValidationUtilsTest.java

```diff
@@ -76,4 +88,64 @@ public void testValidationOfValidInputString() {
     // no exception is thrown
     ValidationUtils.requireNotBlank("abc", "error msg");
   }
+
+  @Test
+  public void testRelationshipValidationDifferentDataType() {
+    RelationshipTermModel fromTerm = new RelationshipTermModel().column("hasCat").table("person");
```

Comment:
I'm having a little difficulty following the tests' intentions -- that is, it's not obvious that this relationship has invalid types.

What do you think about defining a few static String variables for the sample table column names that make the types clear, and using them in relationship construction along with table definition?   Maybe something like:

```
private static final String PERSON_BOOLEAN_COLUMN = "hasCat";
private static final String PERSON_INTEGER_COLUMN = "id";
private static final String CAR_INTEGER_COLUMN = "ownerId";

…

    RelationshipTermModel fromTerm = new RelationshipTermModel().column(PERSON_BOOLEAN_COLUMN).table("person");
    RelationshipTermModel toTerm = new RelationshipTermModel().column(CAR_INTEGER_COLUMN).table("car");
```

Reply:
I think that makes this a lot clearer, thanks for the idea!